app/services/katello/candlepin/event_handler.rb

```diff
@@ -37,31 +37,12 @@ def handle(message)
 
       private
 
-      def subscription_facet
-        message_handler.subscription_facet
+      def event_no_longer_handled
+        @logger.error "Received #{message_handler.subject} event from Candlepin. Handling of this event is no longer supported."
```

Comment:
This is an error, but is it really? Shouldn't it at most be a warning or does it really break something? Also, what's the expected action a user should take to prevent this from showing up?

Reply:
If a user receives this error, it's a bug, not a user error. This code won't be around forever; I intend to remove the actual handling (and these errors) when we do SCA-only cleanup hopefully soon.

Reply:
Closing the loop: now tracked in https://issues.redhat.com/browse/SAT-30448.